### PR TITLE
DOCS-2902: Make machine api edits from qa

### PR DESCRIPTION
**Pull request by sguequierre**

### CODE REVIEW

File: docs/appendix/apis/robot.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 tags: ["robot state", "sdk", "apis", "robot api"]
 aliases:
   - /program/apis/robot/
   - /build/program/apis/robot/
 ---
 
-The _robot API_ is the application programming interface that manages each of your machines running `viam-server`.
-Use the robot API to connect to your machine from within a supported [Viam SDK](/appendix/apis/), and send commands remotely.
+The _machine API_ is the application programming interface that manages each of your machines running `viam-server`.
+Use the machien API to connect to your machine from within a supported [Viam SDK](/appendix/apis/), and send commands remotely.
```
Comment (sguequierre) [inline docs/appendix/apis/robot.md:14]:
```suggestion
Use the machine API to connect to your machine from within a supported [Viam SDK](/appendix/apis/), and send commands remotely.
```

File: docs/appendix/apis/robot.md [hunk 2]
```diff
@@ -32,16 +32,20 @@ from viam.rpc.dial import DialOptions, Credentials
...
+        disable_sessions=True,
+        dial_options=DialOptions(timeout=10)).with_api_key(
+            # Replace "<API-KEY>" (including brackets) with your API key
+            api_key='<API-KEY>',
+            # Replace "<API-KEY-ID>" (including brackets) with your API key
+            # ID
+            api_key_id='<API-KEY-ID>'
+        )
+    return await RobotClient.at_address(
+      address='ADDRESS FROM CODE SAMPLE TAB OF VIAM APP',
```
Comment (sguequierre) [inline docs/appendix/apis/robot.md:46]:
```suggestion
      address='ADDRESS FROM CONNECT TAB OF VIAM APP',
```

File: static/include/robot/apis/generated/robot.md [hunk 3]
```diff
@@ -302,7 +301,24 @@ Transform a given source Pose from the original reference frame to a new destina
...
 
 ```python {class="line-numbers linkable-line-numbers"}
-pose = await machine.transform_pose(PoseInFrame(), "origin")
+from viam.proto.common import Pose, PoseInFrame
+
+pose = Pose(
+    x=1.0,    # X coordinate in mm
+    y=2.0,    # Y coordinate in mm
+    z=3.0,    # Z coordinate in mm
+    o_x=0.0,  # Orientation radian X
```
Comment (sguequierre) [inline static/include/robot/apis/generated/robot.md:310]:
```suggestion
    o_x=0.0,  # Orientation vector X
```

File: static/include/robot/apis/generated/robot.md [hunk 4]
```diff
@@ -302,7 +301,24 @@ Transform a given source Pose from the original reference frame to a new destina
...
+    o_y=0.0,  # Orientation radian Y
```
Comment (sguequierre) [inline static/include/robot/apis/generated/robot.md:311]:
```suggestion
    o_y=1.0,  # Orientation unit vector Y
```

File: static/include/robot/apis/generated/robot.md [hunk 5]
```diff
@@ -302,7 +301,24 @@ Transform a given source Pose from the original reference frame to a new destina
...
+    o_z=0.0,  # Orientation radian Z
```
Comment (sguequierre) [inline static/include/robot/apis/generated/robot.md:312]:
```suggestion
    o_z=0.0,  # Orientation unit vector Z
```

File: static/include/robot/apis/generated/robot.md [hunk 6]
```diff
@@ -302,7 +301,24 @@ Transform a given source Pose from the original reference frame to a new destina
...
+    theta=0.0 # Orientation angle in radians
```
Comment (sguequierre) [inline static/include/robot/apis/generated/robot.md:313]:
```suggestion
    theta=25.0 # Orientation angle in degrees
```

File: static/include/robot/apis/generated/robot.md [hunk 7]
```diff
@@ -302,7 +301,24 @@ Transform a given source Pose from the original reference frame to a new destina
...
-pose = await machine.transform_pose(PoseInFrame(), "origin")
+from viam.proto.common import Pose, PoseInFrame
+
+pose = Pose(
+    x=1.0,    # X coordinate in mm
+    y=2.0,    # Y coordinate in mm
+    z=3.0,    # Z coordinate in mm
+    o_x=0.0,  # Orientation unit vector X
+    o_y=1.0,  # Orientation unit vector Y
+    o_z=0.0,  # Orientation unit vector Z
```
Comment (JessamyT) [inline static/include/robot/apis/generated/robot.md:312]:
```suggestion
    o_x=0.0,  # X component of orientation vector
    o_y=1.0,  # Y component of orientation vector
    o_z=0.0,  # Z component of orientation vector
```

File: static/include/robot/apis/generated/robot.md [hunk 8]
```diff
@@ -406,7 +422,7 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 **Example:**
 
 ```go {class="line-numbers linkable-line-numbers"}
-status, err := machine.Status(ctx)
+status, err := machine.Status(ctx, nil)
```
Comment (npentrel) [inline static/include/robot/apis/generated/robot.md:425]: why ctx here and not context.Background()? I thought we had decided to always do that

Comment (sguequierre) [reply static/include/robot/apis/generated/robot.md:425]: oh context as ctx is everywhere in the docs, I normally use context.Background but thought we had decided to do that (ctx). I saw it in machine management ticket as well but didn't think there was a problem. Could we make a separate ticket to address sweepingly?

File: docs/appendix/apis/robot.md [hunk 9]
```diff
@@ -1,15 +1,16 @@
...
 type: "docs"
-description: "How to use the Robot API to monitor and manage your machines."
+description: "How to use the Machine API to monitor and manage your machines."
 tags: ["robot state", "sdk", "apis", "robot api"]
 aliases:
   - /program/apis/robot/
   - /build/program/apis/robot/
 ---
 
+
```
Comment (sguequierre) [inline docs/appendix/apis/robot.md:13]:
```suggestion
```

File: static/include/robot/apis/generated/robot.md [hunk 10]
```diff
@@ -132,29 +132,31 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
...
+
 # Define a new discovery query.
-q = machine.DiscoveryQuery(subtype=acme.API, model="some model")
+q = DiscoveryQuery(subtype="camera", model="webcam")
 
 # Define a list of discovery queries.
 qs = [q]
 
-# Get component configurations with these queries.
+# # Get component configurations with these queries.
```
Comment (npentrel) [inline static/include/robot/apis/generated/robot.md:159]:
```suggestion
# Get component configurations with these queries.
```

**Review by npentrel — APPROVED** [review]
LGTM % nits

File: static/include/robot/apis/generated/robot.md (hunk 8 above)
Comment (sguequierre) [reply static/include/robot/apis/generated/robot.md:426]: sorry, I was just confused. Will do. @npentrel

**Pull request merged**